### DISCUSSION TITLE
can you add custom ranking

### DISCUSSION THREAD

**Discussion by starz91** [dutchdronesquad/rh-stream-overlays#92] [Ideas]
hi,
can you add an overlay result as default to obtain a personalized ranking ?  i use last heating position plugin 

thank you

![image](https://github.com/user-attachments/assets/0794acf9-bf22-4616-b2cf-dcd17ea59df7)

**Comment by klaasnicolaas** [comment]
Would definitely like to add this, could you share a backup? That would make it a bit easier for me to test with all your custom ranking settings and see what the code does with the data.

./Klaas
